API_doc/LibConfig.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+# include "APILibConfig.h"
+
+namespace LibConfig
+{
+
+  /*
+   * manage your subscriptions
+   */
+  struct Subscription
+  {
+  private:
+    onChangeNotifier subscribedFunc;
+  public:
+
+    Subscription(onChangeNotifier);
+
+    void unsubscribe(); // stop your subscription
```

Comment:
This is a helper header, functions should be declared with their implementation.

Reply:
Plus, if we want to store a functor the class needs to be templated anyway.

Reply:
What is it then? I thought it was supposed to be a fine C++ layer on top of the .h.